src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -272,6 +269,102 @@ export class CollapseCellsAction extends ToggleableAction {
 	}
 }
 
+// Run Notebook with Parameters
+export class RunParametersAction extends TooltipFromLabelAction {
+	private static readonly label = localize('runParameters', "Run with Parameters");
+	private static readonly iconClass = 'icon-run-with-parameters';
+
+	constructor(id: string, toggleTooltip: boolean,
```

Comment:
`toggleTooltip `on its own line? That would fit with the other arguments

Reply:
Sure can do that.